cookbook/product_asset/mass_import.rst

```diff
@@ -0,0 +1,25 @@
+How to Mass Import Assets
+=========================
+
+A command has been created in the PIM to allow you to mass import assets: ``php app/console pim:product-asset:mass-upload``
+
+For this command to work, you need to copy files you want to import into the folder configured in ``pim_parameters.ym`` under the parameter ``tmp_storage``.
+The command will upload the files into the application.
+
+For example, if you upload a file named "demo_video.wmv", Akeneo PIM will check if an asset with the code "demo_video" exists.
```

Comment:
wmv is pretty dead an not open, maybe a open source format ? like ogg for example ? It's a details, not a big deal